include/sys/util.h

```diff
@@ -185,7 +185,7 @@ extern "C" {
  */
 #ifndef CLAMP
 /* Use Z_CLAMP for a GCC-only, single evaluation version */
-#define CLAMP(val, low, high) MIN(MAX(val, low), high)
+#define CLAMP(val, low, high) (((val) <= (low)) ? (low) : MIN(val, high))
```

Comment:
You should amend the "sys: util: Add clamp macro" commit with this change. Similarly, the unit test modification should be done in the commit that introduces the `CLAMP` macro, not in the one that replaces uses of `MIN(MAX())` with it.

Reply:
Of course, sorry. Will do.